.github/workflows/test.yaml

```diff
@@ -73,9 +120,22 @@ jobs:
         with:
           name: ${{ matrix.os }}-${{ matrix.arch }}-libraries
           path: llm/llama.cpp/build
+      - if: ${{ matrix.os == 'ubuntu-latest' }}
+        uses: actions/download-artifact@v4
+        with:
+          name: cuda-11.8.0-libraries
+          path: llm/llama.cpp/build/cuda
+      - if: ${{ matrix.os == 'ubuntu-latest' }}
+        uses: actions/download-artifact@v4
+        with:
+          name: rocm-6.0-libraries
+          path: llm/llama.cpp/build/rocm
       - uses: golangci/golangci-lint-action@v3
   test:
-    needs: generate
+    needs:
+      - generate
+      - generate-cuda
+      - generate-rocm
```

Comment:
We're just running unit tests, so presumably just the CPU generate should be sufficient to keep the embed happy.

Reply:
you're suggesting to generate the gpu libraries but link them in?

Reply:
If the goal is to build a final shippable binary, we need it all, but if the goal is just to build the various pieces to validate no compile error regressions, then we could build the llm variations, but not actually depend on them in this phase of the build to streamline the build.